go.mod

```diff
@@ -0,0 +1,39 @@
+module github.com/aws/amazon-cloudwatch-agent-test
+
+replace github.com/influxdata/telegraf => github.com/aws/telegraf v0.10.2-0.20220502160831-c20ebe67c5ef
```

Comment:
Do we need telegraf here?

Reply:
same as below

Reply:
yeah looks like it works now without telegraf. With agent repo dependency this had to be added